Dockerfile

```
@@ -91,37 +88,39 @@ RUN curl -sSL https://github.com/grpc/grpc-web/releases/download/${grpc_web_vers
     -o /tmp/grpc_web_plugin && \
     chmod +x /tmp/grpc_web_plugin
 
-FROM alpine:$alpine_version AS protoc-all
+FROM debian:$debian-slim AS protoc-all
 
 ARG grpc_version
-ARG proto_version
 
-RUN set -ex && apk --update --no-cache add \
+RUN mkdir -p /usr/share/man/man1
+RUN set -ex && apt-get update && apt-get install -y --no-install-recommends \
     bash \
-    libstdc++ \
-    libc6-compat \
     ca-certificates \
     nodejs \
-    nodejs-npm
-
-RUN set -ex && apk --update --no-cache add \
-    protoc~=${proto_version} \
-    protobuf~=${proto_version} \
-    --repository=http://dl-cdn.alpinelinux.org/alpine/edge/main
-
-RUN set -ex && apk --update --no-cache add \
-    grpc~=${grpc_version} \
-    grpc-cli~=${grpc_version} \
-    --repository=http://dl-cdn.alpinelinux.org/alpine/edge/community
+    npm \
+    zlib1g \
+    libssl1.1 \
+    openjdk-11-jre
 
 # Add TypeScript support
 
 RUN npm config set unsafe-perm true
 RUN npm i -g ts-protoc-gen@0.12.0
 
-COPY --from=build /tmp/grpc-java/compiler/build/exe/java_plugin/protoc-gen-grpc-java /usr/local/bin/
 COPY --from=build /tmp/googleapis/google/ /opt/include/google
 COPY --from=build /tmp/api-common-protos/google/ /opt/include/google
+
+# Copy well known proto files
+COPY --from=build /tmp/grpc/bazel-grpc/external/com_google_protobuf/src/google/protobuf/ /opt/include/google/protobuf/
+# Copy protoc
+COPY --from=build /tmp/grpc/bazel-bin/external/com_google_protobuf/ /usr/local/bin/
```

If it's copying only one binary, shouldn't this be more explicit? e.g.:
```suggestion
COPY --from=build /tmp/grpc/bazel-bin/external/com_google_protobuf/protoc /usr/local/bin/
```

so if something goes wrong we would have an error during building this image, not in runtime.

---

oh, OK, nvm